### PR TITLE
fix(ui) Update flux function docs and examples

### PR DESCRIPTION
**Pull request by sanderson**
Closes #

_Briefly describe your proposed changes:_

  - [x] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [x] Tests pass
  - [x] http/swagger.yml updated (if modified Go structs or API)
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/shared/constants/fluxFunctions.ts [hunk 1]
```diff
@@ -119,36 +123,28 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
...
+    package: '',
     desc: 'Applies an aggregate function to fixed windows of time.',
     example: 'aggregateWindow(every: 1m, fn: mean)',
     category: 'Aggregates',
     link:
-      'https://v2.docs.influxdata.com/v2.0/reference/flux/functions/transformations/aggregates/aggregatewindow',
+      'https://v2.docs.influxdata.com/v2.0/reference/flux/functions/built-in/transformations/aggregates/aggregatewindow/',
   },
   {
-    name: 'assertEquals',
```
Comment (russorat) [inline ui/src/shared/constants/fluxFunctions.ts]: This seems like a strange update. did they really remove the assertEquals function? it's still in the spec: https://github.com/influxdata/flux/blob/master/docs/SPEC.md#assertequals

File: ui/src/shared/constants/fluxFunctions.ts [hunk 2]
```diff
@@ -306,24 +354,25 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
...
         desc: 'The column name for the time values. Defaults to `"_time"`.',
         type: 'String',
       },
     ],
+    package: '',
     desc:
       'Computes the rate of change per unit of time between subsequent non-null records. The output table schema will be the same as the input table.',
     example:
-      'derivative(unit: 1s, nonNegative: true, columns: ["_value"], timeColumn: "_time")',
+      'derivative(unit: 1s, nonNegative: true, column: "_value", timeColumn: "_time")',
```
Comment (russorat) [inline ui/src/shared/constants/fluxFunctions.ts]: the spec still says this is "columns": https://github.com/influxdata/flux/blob/master/docs/SPEC.md#derivative

File: ui/src/shared/constants/fluxFunctions.ts (hunk 1 above)
Comment (sanderson) [reply ui/src/shared/constants/fluxFunctions.ts]: They didn't remove it. It got moved into the `testing` package.

File: ui/src/shared/constants/fluxFunctions.ts [hunk 3]
```diff
@@ -513,9 +581,9 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
         type: 'Integer',
       },
       {
-        name: 'columns',
-        desc: 'List of columns by which to sort. Default is `["_value"]`.',
-        type: 'Array of Strings',
+        name: 'column',
+        desc: 'Column by which to sort. Default is `"_value"`.',
```
Comment (russorat) [inline ui/src/shared/constants/fluxFunctions.ts]: spec still has columns: https://github.com/influxdata/flux/blob/master/docs/SPEC.md#highestlowest

File: ui/src/shared/constants/fluxFunctions.ts (hunk 2 above)
Comment (sanderson) [reply ui/src/shared/constants/fluxFunctions.ts]: You're right. Got over aggressive with my aggregate updates. Will fix this.

File: ui/src/shared/constants/fluxFunctions.ts (hunk 3 above)
Comment (sanderson) [reply ui/src/shared/constants/fluxFunctions.ts]: Looks like the SPEC needs to be updated here. This was updated with all the aggregate functions so they only accept a single column now:
https://github.com/influxdata/flux/blob/928e70566bea61accd45c43ec04b1692f740964e/stdlib/universe/universe.flux#L171

**Pull request closed without merging**